### PR TITLE
Create a database based on a diagram

### PR DESCRIPTION
**Pull request by AxelSoler**
## Team @marurevi @AxelSoler 

- Create a file where we implement the database from the diagram.
- Join tables 
- Add the FK indexes.

image

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    id SERIAL PRIMARY KEY,
+    unit_price decimal,
+    quantity integer,
+    total_price decimal,
+    invoice_id integer,
+    treatment_id integer
+);
+
+CREATE TABLE invoice (
+    id SERIAL PRIMARY KEY,
```
Comment (julie-ify) [inline schema_based_on_diagram.sql:24]: - [x] Kindly change this table name to its plural form `invoices`. This is an active record convention of naming tables.

File: schema_based_on_diagram.sql [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+);
+
+-- Modify medical_history table
+ALTER TABLE medical_histories
+    ADD FOREIGN KEY(patient_id) REFERENCES patients(id);
+
+-- Modify treatments table
+ALTER TABLE treatments
+    ADD FOREIGN KEY(id) REFERENCES medical_histories(id);
+
```
Comment (julie-ify) [inline schema_based_on_diagram.sql:45]: - [x] Good job so far, however, You need to create a "join table" for the many-to-many relationship between the `medical_histories` table and the `treatments` table according to the ERD given to you. Also, create the foreign keys and indexes for the join table.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @AxelSoler and @marurevi,

Good job so far!

## Highlights!
- GitFlow was followed ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

- [x] **[REQUIRED]** Kindly add a descriptive Readme to your repository making use of the Microverse Readme template.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql (hunk 2 above)
Comment (AxelSoler) [reply schema_based_on_diagram.sql:45]: We added the required join table with its respective indexes

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi 👋 @AxelSoler, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- You did well to make the changes requested by the previous reviewer 👍 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**